content/management/changelog/index.mdx

```diff
@@ -49,10 +49,23 @@ Since the migration is still ongoing, different parts will be successively moved
 
 ### Webhooks
 
-- The [**Unregister Webhook**](/management/configuration-api/v3.3/#unregister-webhook) method had the `webhook_id` parameter renamed to `id`.
+- The [**Unregister Webhook**](/management/configuration-api/v3.3/#unregister-webhook) method:
+  - had the `webhook_id` parameter renamed to `id`.
+  - had required scope changed to `webhooks.configuration:rw`.
```

Comment:
```suggestion
  - had the required scope changed to `webhooks.configuration:rw`.
```

Reply:
I wonder if we should mention the scope change - we've never done that.

Reply:
imo it's worth to mention - in this case old scopes are no longer valid for this method